package.json

```diff
@@ -61,10 +61,10 @@
     "stylelint-order": "^0.7.0",
     "stylelint-scss": "^2.0.0",
     "stylelint-suitcss": "^1.0.0",
-    "terra-clinical": "git://github.com/cerner/terra-clinical.git#master",
-    "terra-core": "git://github.com/cerner/terra-core.git#master",
+    "terra-clinical": "git://github.com/cerner/terra-clinical.git#terra-clinical-onset-picker@2.1.0",
+    "terra-core": "git://github.com/cerner/terra-core.git#terra-toggle@2.2.0",
     "terra-dev-site": "^0.2.0",
-    "terra-framework": "git://github.com/cerner/terra-framework.git#master",
+    "terra-framework": "git://github.com/cerner/terra-framework.git#terra-theme-provider@2.1.0",
```

Comment:
This looks a weird, but how lerna works is that it adds tags for all the components released at a point in time. So pointing to just the last released component tag and it will encompass all of them.